src/main/java/com/exasol/releaserobot/AbstractPlatformValidator.java

```diff
@@ -9,6 +11,7 @@
  */
 public abstract class AbstractPlatformValidator implements PlatformValidator {
     protected final GitBranchContent branchContent;
+    public final Report report = new ReportImpl(VALIDATION);
```

Comment:
this one we probably want to remove as well, mostly cause it creates a temporal couple. For example, if you call the validator twice, you'll get different results